server/models/Request.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require("mongoose");
+
+const { Schema, model } = mongoose;
+
+const requestSchema = new mongoose.Schema(
+  {
+    owner: {
+      type: Schema.Types.ObjectId,
+      ref: "User",
+      required: true,
+    },
+    sitter: {
+      type: Schema.Types.ObjectId,
+      ref: "User",
+      required: true,
+    },
+    startDate: {
+      type: Date,
+      required: true,
+    },
+    endDate: {
+      type: Date,
+      required: true,
+    },
+    status: {
+      type: String,
+      enum: ["ACCEPTED", "DECLINED", "PAID"],
```

Comment:
```suggestion
      enum: ["ACCEPTED", "DECLINED", "PAID", "PENDING"],
      required: true,
      default: "PENDING"
```

All the fields in this model will be 'required', please update accordingly :)

Reply:
I noticed that a pending status was missing by doing the request controller PR, so I added it there. 😅 However, I can include it here too as you suggest.